Test-Source/Invoke-DockerBuild.Tests.ps1

```diff
@@ -28,30 +21,54 @@ Describe 'Build docker images' {
         It 'creates correct docker build command' {
             Invoke-DockerBuild -ImageName "leeandrasmus" -Context $Global:DockerImagesDir -Dockerfile $dockerFile
             $result = GetMockValue -Key "command"
-            $result | Should -BeLikeExactly "docker build `"$($Global:DockerImagesDir)`" -t leeandrasmus:latest -f `"${dockerFile}`""
+            $result | Should -BeExactly "docker build `"$Global:DockerImagesDir`" -t leeandrasmus:latest -f `"${dockerFile}`" "
```

Comment:
@zionyx to avoid this change I suggest you adapt the appending code 😄

```powershell
$dockerBuildCommand = "docker build `"${Context}`" -t ${postfixedRegistry}${ImageName}:${Tag} -f `"${Dockerfile}`""
if ($ExtraParams) { $dockerBuildCommand = "${dockerBuildCommand} ${ExtraParams}"
```

Reply:
Done! A little differently but it's something both you @casz and @rasmusjelsgaard should be fine with. 😄